scanner/services/converters/converters.go

```diff
@@ -276,6 +278,30 @@ func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircor
 	return vulnerabilities, nil
 }
 
+// getVulnName returns the first vulnerability name that matches a known
+// vulnerability name pattern from the report, or the original name if none
+// found.
+func getVulnName(vuln *claircore.Vulnerability) string {
+	vulnID := vulnNamePattern.FindString(vuln.Name)
+	if vulnID != "" {
+		return vulnID
+	}
+	found := vulnNamePattern.FindAllString(vuln.Links, -1)
+	// Prioritize RHSA > CVE > anything else.
```

Comment:
why not prioritize RHEA no RHBA over CVE as well? Also, when it comes to Red Hat data, the first link is the one we want, so if the first link is RHBA but the second is CVE, then we will show CVE instead of RHBA

Reply:
> why not prioritize RHEA no RHBA over CVE as well?

Are they supposed to be prioritized that way? I think you are saying they have higher priority instead of actually asking a question. Let me know, we can just change the code to match what needs to be done.

> the first link is the one we want, so if the first link is RHBA but the second is CVE, then we will show CVE instead of RHBA

Not sure we should rely on the link order; More robust to keep our own priority.

Reply:
Yeah let's prioritize RHEA and RHBA over CVE. The reason I bring up the link is because if we did not prioritize RHBA over CVE (for example), then we'd call an RHBA a CVE, which we don't want to do. I think we should also prioritize ALAS* over CVE

Reply:
To clarify: RHSA > RHEA > RHBA > ALAS* > CVE

Reply:
Generally, it seems OK to prioritize ALAS over CVE. But why?